bugsnag-android-core/src/main/java/com/bugsnag/android/Thread.java

```diff
@@ -40,14 +40,15 @@ private void logNull(String property) {
     /**
      * Sets the unique ID of the thread (from {@link java.lang.Thread})
      */
-    public void setId(long id) {
+    public void setId(@NonNull String id) {
         impl.setId(id);
     }
```

Comment:
this hides a `NullPointerException` from the Kotlin side, so we need a `null` check:

```suggestion
    public void setId(@NonNull String id) {
        if (id != null) {
            impl.setId(id);
        } else {
            logNull("id");
        }
    }
```